spec/jobs/batch_create_job_spec.rb

```diff
@@ -65,5 +65,14 @@
         expect(log.reload.status).to eq 'failure'
       end
     end
+
+    context "when multiple resource types are passed" do
+      let(:resource_types) { { upload1.id.to_s => ['Image', 'Text'], upload2.id.to_s => ['Image', 'Text'] } }
+      it "sets them all on the record" do
+        subject
+        expect(GenericWork.last.reload.resource_type).to include 'Image'
```

Comment:
Each call to `GenericWork.last.reload` is hitting Fedora. Please capture in a local variable. I'm wondering if you could assert the expected array?

```ruby
resource_types = GenericWork.last.reload.resource_type
expect(resource_types.sort).to eq(['Image', 'Text'])
```